Arm/Hand/hand.hpp

```diff
@@ -8,15 +8,15 @@ class Hand
   /// The hand has its own MCU that communicates with the arm via UART.
  public:
   // Hand(Uart & uart) : uart_(uart) {}
-  void Initialize(){};
+    void Initialize(){};
 
   struct Fingers
   {
-    float pinky_angle   = 0;
-    float ring_angle    = 0;
-    float middle_angle  = 0;
-    float pointer_angle = 0;
-    float thumb_angle   = 0;
+    float pinky_angle_   = 0;
+    float ring_angle_    = 0;
+    float middle_angle_  = 0;
+    float pointer_angle_ = 0;
+    float thumb_angle_   = 0;
```

Comment:
Leave the trailing underscore off. Here's the [Google style guide reference](https://google.github.io/styleguide/cppguide.html#Variable_Names)